commands/list_test.go

```diff
@@ -65,3 +65,47 @@ Agent type                  Status     Agent ID
 		})
 	}
 }
+
+func TestNiceAgentStatus(t *testing.T) {
+	type fields struct {
+		AgentType string
```

Comment:
What's the reason to add fields which are not required in current test?

Reply:
You are right. I forgot delete it.